presto-hive/src/test/java/io/prestosql/plugin/hive/TestBackgroundHiveSplitLoader.java

```diff
@@ -400,7 +400,7 @@ public void testCachedDirectoryLister()
         CachingDirectoryLister cachingDirectoryLister = new CachingDirectoryLister(new Duration(5, TimeUnit.MINUTES), 1000, ImmutableList.of("test_dbname.test_table"));
         assertEquals(cachingDirectoryLister.getRequestCount(), 0);
 
-        int totalCount = 1000;
+        int totalCount = 100;
```

Comment:
where did original number come from. Is 100 enough?

Reply:
that what i infer from how the assertions are structured

https://github.com/prestosql/presto/blob/77886859578094f0b1828725bd60587881497124/presto-hive/src/test/java/io/prestosql/plugin/hive/TestBackgroundHiveSplitLoader.java#L447-L449